guides/common/modules/ref_dhcp-options-for-network-configuration.adoc

```diff
@@ -27,4 +27,8 @@ Set this to the address for {ProjectServer} on `eth1`.
 --foreman-proxy-dhcp-server::
```

Comment:
I think this whole reference section needs redesign. It all only applies to the `dhcp_nsupdate` provider (except `--foreman-proxy-dhcp`), but the section doesn't mention that at all. Not a blocker for this PR, but this PR opens the path to make that sensible.

Reply:
Leaving this as is for now.

Reply:
Yes, that's why I suggested to leave it out of the Satellite guide. Then it's safer for a cherry pick.

Reply:
I have now hidden it for build target satellite.